grommet/hpe-design-system
# Update Forms page examples to include aria labels

[Deploy Preview](https://deploy-preview-2743--keen-mayer-a86c8b.netlify.app/templates)
Note: navigating to templates/forms directly shows "client side error" unrelated to this PR. 
![image](https://user-images.githubusercontent.com/23144209/179621780-22277133-2404-48cf-8c32-09cde6820578.png)
Workaround: go to current Preview and click on "Forms" card.


#### What does this PR do?
Addresses the accessibility violations (errors) within examples on the Forms page. Removes the placeholder text in text inputs in favor of labels. Orphaned aria labels remain as there is no standardized way to correctly label them, to be updated in the future.

#### Where should the reviewer start?
aries-site/src/examples/templates/forms:
- ShippingExample.js
- RequiredFieldsExample.js
- SettingsExample.js
- SignUpExample.js
- SortExample.js

#### What testing has been done on this PR?
Manual visual testing using WAVE accessibility tool for accessibility violations

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**

**Accessibility Checks**

**Code Quality Checks**

#### How should this be manually tested?
Run WAVE accessibility tests to ensure no more accessibility violations in Forms code examples.

#### Any background context you want to provide?
The remaining WAVE violation relates to an site-wide search feature not maintained by the DS. This is the only error remaining on the page.

#### What are the relevant issues?
Closes #2735.

#### Screenshots (if appropriate)
<img width="1173" alt="image" src="https://user-images.githubusercontent.com/23144209/179619859-1f484784-5147-4fd7-b58a-64ce16969b44.png">
<img width="666" alt="image" src="https://user-images.githubusercontent.com/23144209/179619887-0da00601-bff1-41b9-a123-3b988e755c38.png">


#### Should this PR be mentioned in Design System updates?
No - just a small accessibility loose end.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.
